### DISCUSSION TITLE
Pages with Parameter?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#7813] [Help]
How can i create a page with accessing on paramter like 'custompage/1'.
When i create a page, i can only access it on "/custompage", with ".../1" i get, page not found.

**Comment by danharrin** [comment]
Have you defined the route with a parameter?

**Comment by leandrocfe** [comment]
same question here
https://github.com/filamentphp/filament/discussions/7819

**Comment by robertpotkowa** [comment]
Ohh, that works! Thx

**Comment by RPotkowaVegasystems** [comment]
But i got now an another bug/error: I have on my page "/custmpage/id" a livewire component inside, that creates a filament table with actions like create/edit. The form any everything works, but when i  hit the save button, i get an error: 

Unable to find component: [filament.livewire.notifications]

When i remove the route from web and access the page by "/custompage" without the ID, it works, but my problem is, that i need to pass the ID to the page.
